### ISSUE TITLE
Investigate a simple method to detach beans

### ISSUE THREAD

**Issue by kenwenzel** [komma/komma#23]
Users often want to use detached beans as DTOs between backend and UI. It would help if KOMMA could support this use case by generating the DTOs in the same way as beans are generated.

Some ideas:
- Provide method on a bean `bean.detach()` or `bean.snapshot():
  - How does the bean know which properties should be copied into the new POJO? Are the relevant properties are already intialized or does this method load them from the store?
- Provide a method on `IEntityManager` like `findDetached(reference)` or even on a query like `evaluateDetached()`:
  - KOMMA already supports prefetching via CONSTRUCT queries. So this could be a query-driven approach to fetch exactly those properties that should be present in a detached bean.
